spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/MigrateTableProcedure.java

```diff
@@ -39,7 +39,8 @@ class MigrateTableProcedure extends BaseProcedure {
       new ProcedureParameter[] {
         ProcedureParameter.required("table", DataTypes.StringType),
         ProcedureParameter.optional("properties", STRING_MAP),
-        ProcedureParameter.optional("drop_backup", DataTypes.BooleanType)
+        ProcedureParameter.optional("drop_backup", DataTypes.BooleanType),
+        ProcedureParameter.optional("max_concurrent_read_datafiles", DataTypes.IntegerType)
```

Comment:
Any reason we can't just call this parameter "parallelism" ? I think it will only be data files involved

Reply:
Thanks for your reviews, It because this  https://github.com/apache/iceberg/pull/3973#discussion_r794150006

Reply:
@amogh-jahagirdar Another reason is that `importSparkPartitions`  https://github.com/kingeasternsun/iceberg/blob/feature/add-parallelism-add-files/spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java#L620-L630 has a local variable named `parallelism`

Reply:
@amogh-jahagirdar ,Thanks for your review,  I have fixed it.